src/index.js

```diff
@@ -169,7 +172,7 @@ export default function renderToString(vnode, context, opts, inner) {
 		for (let i=0; i<len; i++) {
 			let child = children[i];
 			if (!falsey(child)) {
-				let ret = renderToString(child, context, opts, true);
+				let ret = renderToString(child, context, opts, true, nodeName==='svg');
```

Comment:
Also, would you prefer I changed `isSvgMode` to be a global flag like in preact/diff rather than an argument to `renderToString`?